job-descriptions/software-engineer-security.md

```diff
@@ -0,0 +1,51 @@
+![logo](https://sourcegraph.com/.assets/img/sourcegraph-light-head-logo.svg)
+
+# Software Engineer - Security
+
+We are looking for a software engineer to own the security of our product. This includes the security of sourcegraph.com as well as making it easy for our customers to secure their own private Sourcegraph instances that they deploy on-premise. You will be responsible for proactively understanding, documenting, and improving the security of our product, as well as promptly responding to and addressing security issues reported to us by third parties. As our security needs grow over time, there is a potential for you to grow into a security team leadership role.
+
+## Qualifications
+
+We are looking for candidates with the following skills, and we will test for these things in our interview process:
+
+- Broad knowledge of possible attack vectors for applications like Sourcegraph and an understanding of how to eliminate and/or mitigate those security risks.
+- Skilled at understanding, debugging, and writing Go and TypeScript to identify and fix security issues.
+- Demonstrated ability to own large ambiguous problem areas while making continuous progress.
+- Ability to communicate clearly and empathetically, especially in writing and documentation.
+
+## Nice-to-haves
+
+- Published blog posts and/or tech talks about your work.
+- Experience working on high-performing teams, preferably tech startups.
+
+## Learn more about us
+
+Sourcegraph is an equal opportunity workplace; we welcome people from all backgrounds and communities.
+
+To create a product that serves the needs of all developers, we are building a diverse [all-remote team](https://about.sourcegraph.com/company/remote) that is [distributed across the world](https://about.sourcegraph.com/company/team).
+
+We provide [competitive compensation](https://about.sourcegraph.com/handbook/people-ops/compensation) and [practical benefits](https://about.sourcegraph.com/handbook/people-ops/benefits-and-perks) to keep you happy, healthy, and productive.
+
+Learn more about what it is like to work at Sourcegraph by reading [our handbook](https://about.sourcegraph.com/handbook/).
+
+## Interview process
+
+1. You [apply here](TODO).
+1. We set up a 30 minute call to answer any questions that you have about Sourcegraph.
+1. We evaluate relevant technical skills that you have via an asynchronous coding exercise.
```

Comment:
Depending on the discussion of the question I asked above, the structure of the interview process may need changes.

Reply:
I don't expect the structure to change (but some of the particular questions we ask in these modules might). This person still needs to be able to code (code exercise), they still need to be able to reason about high level architecture (security related architecture question), they still need to be able to have solid technical communication (technical experience), and they still need to be able to work on a team (team collaboration).

Reply:
How can we evaluate general security expertise without additional changes to the interviews?

Reply:
By asking a security related architecture question, by hearing them teach/explain security related work they have done in the past in the technical experience interview, and, if necessary, creating a different coding question that is more security related.

Reply:
> if necessary, creating a different coding question that is more security related.

My intuition is that doing this is necessary to get a good enough signal.